src/main/java/frc/robot/subsystems/arm/PrototypeArm.java

```diff
@@ -0,0 +1,145 @@
+    public final TalonSRX elbowMotor = new TalonSRX(Ports.prototypeArmPorts.ELBOW_MOTOR);
+    public final UnitModel unitModel = new UnitModel(ArmConstants.TICKS_PER_RADIAN);
+
+    private double prevShoulderVelocity;
+    private double shoulderFeedforward;
+    private double prevElbowVelocity;
+    private double elbowFeedForward;
+    private double Time2;
+
+    public PrototypeArm() {
+        super(new PrototypeArmLogInputs());
+        timer.reset();
+        timer.start();
+
+        shoulderMotor.configVoltageCompSaturation(ArmConstants.CONFIG_VOLT_COMP);
+        shoulderMotor.enableVoltageCompensation(ArmConstants.ENABLE_VOLT_COMPANSATION);
+        shoulderMotor.setNeutralMode(NeutralMode.Brake);
+        shoulderMotor.setInverted(ArmConstants.clockWise);
+        shoulderMotor.config_kP(0, ArmConstants.shoulderP);
+        shoulderMotor.config_kI(0, ArmConstants.shoulderI);
+        shoulderMotor.config_kD(0, ArmConstants.shoulderD);
+//        shoulderMotor.configSelectedFeedbackSensor(TalonSRXFeedbackDevice.Analog, 0, 20);
+
+
+        elbowMotor.configVoltageCompSaturation(ArmConstants.CONFIG_VOLT_COMP);
+        elbowMotor.enableVoltageCompensation(ArmConstants.ENABLE_VOLT_COMPANSATION);
+        elbowMotor.setNeutralMode(NeutralMode.Brake);
+        elbowMotor.setInverted(ArmConstants.clockWise);
+        elbowMotor.config_kP(0, ArmConstants.elbowP);
+        elbowMotor.config_kI(0, ArmConstants.elbowI);
+        elbowMotor.config_kD(0, ArmConstants.elbowD);
+
+    }
+
+    public void setShoulderJointPower(double power) {
+        shoulderMotor.set(TalonSRXControlMode.PercentOutput, power);
+    }
+
+    public void setElbowJointPower(double power) {
+        elbowMotor.set(TalonSRXControlMode.PercentOutput, power);
+    }
+
+    public void setShoulderJointVelocity(double velocity){
+        shoulderMotor.set(ControlMode.Velocity, velocity);
+    }
+
+    public void setElbowJointVelocity(double velocity){
+        elbowMotor.set(ControlMode.Velocity, velocity);
+    }
+
+    public double getShoulderJointPosition() {
+        return Math.toDegrees(unitModel.toUnits(shoulderMotor.getSelectedSensorPosition()));
+    }
+
+    public void setShoulderJointPosition(double angle) {
+        shoulderMotor.set(ControlMode.MotionMagic, unitModel.toTicks(Math.toRadians(angle)), DemandType.ArbitraryFeedForward, shoulderFeedforward);
+    }
+
+    public double getElbowJointPosition() {
+        return Math.toDegrees(unitModel.toUnits(elbowMotor.getSelectedSensorPosition()));
+    }
+
+    public void setElbowJointPosition(double angle) {
+        elbowMotor.set(ControlMode.MotionMagic, unitModel.toTicks(Math.toRadians(angle)), DemandType.ArbitraryFeedForward, elbowFeedForward);
+    }
+
+
+    public double getShoulderMotorPower() {
+        return shoulderMotor.getMotorOutputPercent();
+    }
+
+    public double getElbowMotorPower() {
+        return elbowMotor.getMotorOutputPercent();
+    }
```

Comment:
What is a use case for these functions?

Reply:
It's used in the commands to set the motor power to zero when the command is finished.

Reply:
They're used for the logger

Reply:
IMO creating a function, that is used only once, and for this purpose, is a bit overkill.
Writing `loggerInputs.elbowMotorPower = elbowMotor.getMotorOutputPercent()` in the `updateInputs` LGTM.